src/libraries/System.Net.Http/src/System/Net/Http/Headers/CacheControlHeaderValue.cs

```diff
@@ -367,14 +367,14 @@ public static CacheControlHeaderValue Parse(string? input)
             return (CacheControlHeaderValue)CacheControlHeaderParser.Parser.ParseValue(input, null, ref index);
         }
 
-        public static bool TryParse(string? input, out CacheControlHeaderValue? parsedValue)
+        public static bool TryParse(string? input, out CacheControlHeaderValue parsedValue)
```

Comment:
```suggestion
        public static bool TryParse(string? input, [NotNullWhen(true)] out CacheControlHeaderValue parsedValue)
```

The value should still be nullable, it just needs the annotation that it won't be null if the method returns true.

The reference assembly also has to be updated to match: https://github.com/dotnet/runtime/blob/main/src/libraries/System.Net.Http/ref/System.Net.Http.cs